docs/_extra/api/hypothesis.yaml

```diff
@@ -131,8 +131,18 @@ paths:
           description: Annotation not found or no permission to view
           schema:
             $ref: '#/definitions/Error'
-    put:
+    patch:
       summary: Update an annotation
+      description: |
+        This endpoint is available under both the `PATCH` and `PUT`
+        request methods. Both endpoints have PATCH-characteristics
```

Comment:
<GRAMMAR-TROLL>It may solve both your concerns to know that a hyphen is probably unnecessary here.</GRAMMAR-TROLL>